app/models/event.py

```diff
@@ -21,6 +21,7 @@ class Event(baseModel):
     recurringId = IntegerField(null=True)
     contactEmail = CharField(null=True)
     contactName = CharField(null=True)
+    program_id = ForeignKeyField(Program, null= True)
```

Comment:
Look at how we name all of our other foreign keys. We don't use `_id` in the peewee model, so that we can reference our program object with `event.program`. Using `event.program_id` is confusing because it's not an ID, it's a whole object in peewee.